model/model.wl

```diff
@@ -81,6 +81,18 @@ pS0 = 1-(pC0 + pH0);
       
 today=QuantityMagnitude[DateDifference[DateList[{2020,1,1}],Today]];
 
+getEst[importLength_, initialInfectionImpulse_, importtime_] := 
+	Module[{totalInfection, sig, mu, areaUnderCurve},
+		totalInfection = importLength*Exp[-initialInfectionImpulse];
+		(* the normal should hit the boundaries at +/- 3 standard deviations *)
+		sig = importLength / 6;
+		mu = importtime + .5*importLength;
+		(* area under gaussian with standard_deviation = sigma *)
+		areaUnderCurve = sig*Sqrt[2*Pi];
+		With[{mu0 = mu, sig0 = sig, ti = totalInfection, auc = areaUnderCurve},
+			Function[t, Piecewise[{{ti/auc * PDF[NormalDistribution[mu0, sig0], t],importtime <= t <= importtime + importLength}} ]]]]
```

Comment:
I added this piecewise term to try and avoid an underflow from evaluating the gaussian far away from its center point.  This doesn't seem to be working though.